src/Nethermind/Nethermind.Core.Test/Encoding/DepositDecoderTests.cs

```diff
@@ -0,0 +1,71 @@
+// SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+using FluentAssertions;
+using Nethermind.Core.ConsensusRequests;
+using Nethermind.Core.Extensions;
+using Nethermind.Evm.Tracing.GethStyle.Custom.JavaScript;
+using Nethermind.Serialization.Rlp;
+using NUnit.Framework;
+
+namespace Nethermind.Core.Test.Encoding;
+
+public class DepositDecoderTests
+{
+    [Test]
+    public void Should_decode()
```

Comment:
Is this class still needed @rjnrohit ? Seems to be duplicated with ConsensusRequestsDecoder?